operations/commit_queue.go

```diff
@@ -57,6 +57,7 @@ func listQueue() cli.Command {
 		Usage: "list the contents of a project's commit queue",
 		Flags: addProjectFlag(),
 		Before: mergeBeforeFuncs(
+			autoUpdateCLI,
```

Comment:
Can we actually only do this for some functions? As opposed to all functions? Deleting a commit queue item for example, people might not want that to take a long time. And listing. Maybe we should only have autoUpdate be for some of the more common commands (evergreen commit-queue merge, evergreen patch, evergreen validate) Or maybe let's just remove it for some of the ones that should be fast (like this deleting item one) and drop a comment saying why it's not included (so we don't think it's an oversight)

Reply:
Sounds good, I've kept it for most patch and commit-queue functions that don't just delete something, and for quick to return functions like lists and version. Added a comment above autoUpdateCLI to explain the rationale.